mypy.ini

```diff
@@ -41,7 +41,7 @@ files =
 #
 # `exclude` is a regex, not a list of paths like `files` (sigh)
 #
-exclude = torch/include/|torch/csrc/|torch/distributed/elastic/agent/server/api.py|torch/testing/_internal|torch/distributed/fsdp/fully_sharded_data_parallel.py
+exclude = torch/include/|torch/csrc/|torch/distributed/elastic/agent/server/api.py|torch/testing/_internal|torch/distributed/fsdp/fully_sharded_data_parallel.py|torch/_refs|torch/_prims
```

Comment:
Since we're greenfielding this code, can we also get type checking on it as well?

Reply:
Yeah I'll fix it in an update but we want to unblock development

Reply:
I think I've said this to you before, but type checking is something that is best added to a development as early as possible, since it precludes you from using certain idioms and you end up structuring your code differently if you want strict typing